devices/flash-imx6ull/nor/nor.c

```diff
@@ -82,7 +82,7 @@ static int nor_readID(qspi_t *qspi, u8 port, u32 *retValue, time_t timeout)
 }
 
 
-__attribute__((section(".noxip"))) int nor_readStatus(qspi_t *qspi, u8 port, u8 *statusByte, time_t timeout)
+int nor_readStatus(qspi_t *qspi, u8 port, u8 *statusByte, time_t timeout)
```

Comment:
not sure: should developer know we're not using XIP on 6ull or should we always mark all functions used in flash access as `noxip`? (eg. XIP is possible on `zynq7000`, currently we're not supporting/using it)

Reply:
In my opinion we either should provide XIP support fully or not at all, being in the middle causes biggest confusion. If `.noxip` was to be used on `imx6ull` then we should also mark all interrupt handlers as `.noxip` and all function called by those function.

Reply:
Yeah, for me the easiest for developer would be to have a single rule eg. "mark all code used in interruptions and NVM storage as noxip" (without the necessity to know whether we can/currently use XIP on that platform or some more confusing rules (eg. common code for arm7a needs to be noxip because of zynq, but imx6ull-specific code can be all xip (as XIP is not supported by this CPU))).

The decision to run from flash or  from RAM is then done only in linker script (right now we do it as `plo` and `plo-ram`, but might make per-project ld scripts in the future).

This is obviously a separate discussion from this PR, just voicing my thoughts.

Reply:
Also it would be great to have some static analysis tools that would catch `.noxip` code calling `.text`